tests/test_long_text_writer.py

```diff
@@ -0,0 +1,36 @@
+import pygame
+import unittest
+from unittest.mock import MagicMock
+from long_text_writer import LongTextWriter
+
+class TestLongTextWriter(unittest.TestCase):
+    def setUp(self) -> None:
+        # pygame.font.init()
```

Comment:
It would probably be good to get rid of this commented out line

Reply:
Right.  Thanks, I'll do that.